src/telegraf.ts

```diff
@@ -116,17 +117,28 @@ export class Telegraf<
 
   get webhookReply() {
     return this.telegram.webhookReply
-  } /* eslint brace-style: 0 */
+  }
 
   catch(handler: (err: any, ctx: TContext) => void) {
     this.handleError = handler
     return this
   }
 
   webhookCallback(path = '/') {
+    let mustFetchBotInfo = !this.botInfo
     return generateCallback(
       path,
-      (update: tt.Update, res: any) => this.handleUpdate(update, res),
+      async (update: tt.Update, res: http.ServerResponse) => {
+        if (mustFetchBotInfo) {
+          mustFetchBotInfo = false
```

Comment:
Not good as well. Now subsequent updates would be processed immiediately, without waiting for `botInfo`. They should ideally await the same promise.

Reply:
Actually, I wouldn't mind just a naive `this.botInfo ??= await this.telegram.getMe()` in this PR, it can be improved upon in the future if it ever causes any issues.

Reply:
It's not too difficult to make all requests await the same promise at this point. Check out 6dc7745

Reply:
I believe that all incoming requests would throw the same error now, it `getMe` fails for some reason. I'm not sure if this should be designed differently. I don't really like it, but I see no other way to make all waiting requests fail other than by making them throw.